composer.json

```diff
@@ -17,35 +17,36 @@
     "rdkafka"
   ],
   "require": {
-    "php": "^8.0",
-    "ext-rdkafka": "^4.0|^5.0|^6.0",
-    "ext-json": "*"
+    "php": "^8.1",
+    "ext-rdkafka": "^4.0|^5.0|^6.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "^9.4",
-    "squizlabs/php_codesniffer": "^3.5.4",
-    "phpstan/phpstan": "^1.8",
-    "php-mock/php-mock-phpunit": "^2.6",
-    "kwn/php-rdkafka-stubs": "^2.0.0",
+    "ext-pcntl": "*",
+    "flix-tech/avro-serde-php": "^1.7.2 || ^2.2.0 || ^3.0.0",
```

Comment:
```json
"flix-tech/avro-serde-php": "^1.7.2"
```
can work only with php `^7.3||^8.0`, meaning can't be shipped alongside php dependency php `8.1`. 

If problem you trying to solve is to use `"flix-tech/avro-serde-php": "^3.0"`, than only thing that's necessary for this pr is 
```json
"flix-tech/avro-serde-php": "^1.7.2 || ^2.2.0 || ^3.0.0",
```
rest can be reverted  and we'll release patch version

Reply:
yup including 8.0. But one of change in this pr is to remove support for 8.0 which is not necessary

Reply:
preferably if it's possible only this
```json
"flix-tech/avro-serde-php": "^1.7.2 || ^2.2.0 || ^3.0.0",
```

Reply:
> but not sure why we need RCs

The reason for releasing RC is so we can go further with development, even-thought  there's still things we might include in next major release.  RC allowing us to incrementally change code without frequent releasing major versions.

Let's have call tomorrow, since I'm bit confused